packages/core/src/app/shipping/stripeUPE/StripeShippingForm.test.tsx

```diff
@@ -83,7 +83,7 @@ describe('StripeShippingForm', () => {
         const { container } = renderContainer({ isLoading: false });
 
         expect(defaultProps.initialize).toHaveBeenCalled();
-        expect(defaultProps.getFields).toHaveBeenCalledTimes(3);
+        expect(defaultProps.getFields).toHaveBeenCalledTimes(2);
```

Comment:
The removed 1 call to `defaultProps.getFields`  is the one in `validationSchema` as it is not called at the time of rendering form any more.